src/types.ts

```diff
@@ -45,3 +45,7 @@ export type AntCardProps = {
 export type ConnectWalletModalProps = {
   setShowModal: any;
 };
+
+export type ArnsCardProps = {
+  name: string;
+};
```

Comment:
```suggestion
```
use the same type we're using for the AntCardProps, can be renamed to be more generic (ArNSMapping)